src/android/app/src/main/jni/native.cpp

```diff
@@ -507,6 +523,27 @@ int Java_org_yuzu_yuzu_1emu_NativeLibrary_installFileToNand(JNIEnv* env, jobject
                                                              GetJString(env, j_file_extension));
 }
 
+jboolean Java_org_yuzu_yuzu_1emu_NativeLibrary_doesUpdateMatchProgram(JNIEnv* env, jobject jobj,
+                                                                      jstring jprogramId,
+                                                                      jstring jupdatePath) {
+    u64 programId = EmulationSession::getProgramId(env, jprogramId);
```

Comment:
```suggestion
    u64 program_id = EmulationSession::getProgramId(env, jprogramId);
```
dito with other variable names in this file down below